Source/com/drew/metadata/exif/ExifReader.java

```diff
@@ -73,23 +73,11 @@ public void readJpegSegments(@NotNull final Iterable<byte[]> segments, @NotNull
             // Filter any segments containing unexpected preambles
             if (segmentBytes.length < JPEG_SEGMENT_PREAMBLE.length() || !new String(segmentBytes, 0, JPEG_SEGMENT_PREAMBLE.length()).equals(JPEG_SEGMENT_PREAMBLE))
                 continue;
-            extract(new ByteArrayReader(segmentBytes), metadata, JPEG_SEGMENT_PREAMBLE.length());
+            extract(new ByteArrayReader(segmentBytes), metadata, JPEG_SEGMENT_PREAMBLE.length(), null);
         }
     }
 
-    /** Reads TIFF formatted Exif data from start of the specified {@link RandomAccessReader}. */
-    public void extract(@NotNull final RandomAccessReader reader, @NotNull final Metadata metadata)
```

Comment:
Is there a good reason to remove these overloads? Java doesn't have optional parameters, so we use overloads instead. Ideally we shouldn't remove APIs people might be using.